exercises/classes/.gitignore

```diff
@@ -0,0 +1 @@
+classes classes_sol
```

Comment:
The other executables are all grouped in the top level .gitignore. I must say I like the locality we have here. I did not know we can have .gitignore files at all levels. Maybe we should move to this everywhere ? What do others think ?

Reply:
Another question : shouldn't you have 2 lines ? Would explain why you pushed classes_sol

Reply:
> Another question : shouldn't you have 2 lines ? Would explain why you pushed classes_sol

Sorry, I check that.